src/main/steps/applicant1/hub-page/sole/latest-update-content/final-order-requested.njk

```diff
@@ -1,3 +1,8 @@
-{% for property, line in finalOrderRequested %}
-  <p class="govuk-body">{{ line }}</p>
-{% endfor %}
+{% if userCase.applicant1AppliedForFinalOrderFirst === 'Yes' %}
```

Comment:
Might be best to have this condition in the content file and pass it here that way you can also use the values from the definition file for 'Yes'